src/install/windows-shim/bun_shim_impl.zig

```diff
@@ -795,8 +795,8 @@ fn launcher(comptime mode: LauncherMode, bun_ctx: anytype) mode.RetType() {
             null,
             null,
             1, // true
-            0,
-            null,
+            w.CREATE_UNICODE_ENVIRONMENT,
+            @constCast(bun_ctx.environment),
```

Comment:
```suggestion
            if (is_standalone) null else @constCast(bun_ctx.environment),
```

In addition, I think `@constCast` might be a subtle mistake.